pkgs/development/interpreters/ruby/default.nix

```diff
@@ -108,6 +108,8 @@ let
           sed -i configure.ac -e '/config.guess/d'
           cp --remove-destination ${config}/config.guess tool/
           cp --remove-destination ${config}/config.sub tool/
+          # Make the build reproducible for ruby <= 2.7
```

Comment:
Maybe add a link to the line on GitHub? Makes retracing this line easier.

https://github.com/ruby/io-console/blob/928125107ae6c167e61edb1c24e0880cb8c52d1f/io-console.gemspec#L8